ui-v2/app/styles/components/healthchecked-resource.scss

```diff
@@ -85,12 +83,25 @@
 .healthy .healthchecked-resource li a {
   padding-left: 0;
 }
+%healthchecked-resource header strong {
+  top: 2.8em;
+  left: 15px;
+}
+%healthchecked-resource header span {
+  margin-bottom: 1.75em;
+}
 %healthchecked-resource header a {
   padding: 12px 15px;
 }
+.unhealthy .healthchecked-resource header a {
+  padding-bottom: 15px;
+}
+.unhealthy .healthchecked-resource header.with-service a {
+  padding-bottom: 25px;
+}
```

Comment:
Why don't these use placeholder selectors? Doesn't that make these selectors dependent on the name of the class that extends the placeholder and thus sorta defeat the purpose of the placeholder?

Reply:
Yeah defo 👍  I'll be splitting them into something like `%healthchecked-resource`, `%unhealthy-resource` and `%healthy-resource` I think. Until yesterday I was in 'tweak-mode', so this will get revisited as part of a larger CSS refactor now I'm back to more 'heavy-handed mode'